src/Features/VisualBasic/Portable/Completion/CompletionProviders/ImportCompletionProvider/ImportCompletionProviderHelper.vb

```diff
@@ -33,6 +34,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
         Public Shared Async Function CreateContextAsync(document As Document, position As Integer, cancellationToken As CancellationToken) As Task(Of SyntaxContext)
             ' Need regular semantic model because we will use it to get imported namespace symbols. Otherwise we will try to 
             ' reach outside of the span And ended up with "node not within syntax tree" error from the speculative model.
+            Dim sourceText = Await document.GetTextAsync(cancellationToken).ConfigureAwait(False)
+            Dim frozenDocument = sourceText.GetDocumentWithFrozenPartialSemantics(cancellationToken)
+            If frozenDocument Is Nothing Then
+                document = frozenDocument
+            End If
```

Comment:
if you wanted `??` equivalent, it's `if(x, y)` in VB.  but we def don't want this in either event :)